tokio-tls/src/lib.rs

```diff
@@ -77,10 +78,14 @@ where
     AllowStd<S>: Read + Write,
 {
     fn drop(&mut self) {
-        (self.0).0.get_mut().context = 0 as *mut ();
+        (self.0).0.get_mut().context = null_mut();
     }
 }
 
+// *mut () context is neither Send nor Sync
+unsafe impl<S> Send for AllowStd<S> {}
```

Comment:
Should these be `unsafe impl<S: Send> Send for AllowStd<S>` and `unsafe impl<S: Sync> Sync for AllowStd<S>` rather than unbounded?

Reply:
Woops, yes, you are correct....

Reply:
sorry missed it, updated!